org by committment level, adding new ideas (#230)
USRSE/usrse.github.io: pages/get_involved.md

subtitle: Ways to contribute to the US-RSE Association
---

The US-RSE Association is a new organization that needs community involvement to grow and mature.  

> Most of our interaction is via Slack: usrse.slack.com.  You'll receive a link to join the Slack space when you [join US-RSE](https://us-rse.org/join/).  If have already joined US-RSE but can't get into Slack, email contact@us-rse.org for help.

Reviewer: Missing word, third sentence should be:  If *you* have already joined...

The US-RSE Association is a new organization that needs community involvement to grow and mature.  Below are some ways members can get more involved in building this organization. Have additional ideas for a working group or committee?  Let us know and bring the discussion to slack.

If you have another idea but aren’t sure how to get started, the Steering Committee would be happy to help. Reach out on Slack (@sc) or email us all at [contact@us-rse.org](mailto:contact@us-rse.org).

<br>

> # Easy

Reviewer 2: ```suggestion
# Quick Contributions
```

---

### Invite a New Member

Email at least one other person who might be interested in the US-RSE association and invite them to join (via the link on the main website page: [https://us-rse.org/](https://us-rse.org/)).  

### Introduce Yourself

Have you been lurking on Slack for a while?  Introduce yourself on the #introductions channel, even if you’re not new to the group.

### Welcome New Members

Join the #introductions channel on Slack and welcome new members.  

### Share a Resource/Ask a Question

> Found a good resource that’s worth sharing? Post it on Slack! Looking for a good resource or some advice?  Post it on Slack!  Looking for a little more guidance? See How to Share with the Community

Reviewer 2: ```suggestion
Whether you've found a good resource that’s worth sharing, or are looking for some advice, you are encouraged to post your ideas and questions on Slack!  Visit https://usrse.slack.com/, or reach out to contact@us-rse.org for an invitation.
```

Author: I like the first sentence, but we want people to join via the form, so I'm going to reword to that part.

<br>

> # One-Time Commitment

Reviewer 2: Just so the headers are clear in what they are describing, I changed the first to Quick contribution, and then it sort of followed to have the others be in similar format (One-time contribution, and Ongoing or Extended contribution)
```suggestion
# One-Time Contribution
```

* **Get involved**: Join the #socialmedia channel on Slack and share your idea or volunteer to tweet

### Organize a Community Call

We are aiming to have community calls at least quarterly, but we need help planning and organizing them.  Can you take the lead organizing an upcoming call?  

* **Get involved**: Join the #communitycalls channel on Slack

### Volunteer to be an RSE Story

Willing to share your story of being an RSE on the [RSE Stories podcast](https://us-rse.org/rse-stories/)? 

* **Get involved**: See [https://us-rse.org/rse-stories/](https://us-rse.org/rse-stories/) and sign-up to be a subject.

### Propose a Conference Event

Going to a conference related to RSEs?  Propose a talk, panel, BoF, or other event.  Members of the Steering Committee are happy to help you with this (contact @sc on Slack).  

* **Get involved**: Raise the idea on Slack to find collaborators

<br>

> # Ongoing

Reviewer 2: ```suggestion
# Ongoing Contribution
```
